### DISCUSSION TITLE
Added a relation, did a rename of column and delete the relation [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2096] [Discord Threads]
> 🙋‍♂️ **alfomin**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### is it an external database ?  
#  
> 🙋‍♂️ **alfomin**  
#### yes   
#### it's a PostgreSQL  
#  
> 💁 **o1lab**  
#### you could either do metadata sync or recreate a new project connecting to that database  
#  
> 💁 **o1lab**  
#### can u pls paste project info  
#  
> 💁 **o1lab**  
#### right top menu  
#  
> 🙋‍♂️ **alfomin**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **pg**   
#### ProjectOnRootDB: **false**   
#### RootDB: **pg**   
#### PackageVersion: **0.90.11**  
#  
> 💁 **o1lab**  
#### ya should be good  
#  
> 💁 **o1lab**  
#### Create a new project by connecting to same db  
#  
> 🙋‍♂️ **alfomin**  
#### ok. will try    
#### thanks!  
#  
> 🙋‍♂️ **alfomin**  
#### all worked well   
####    
#### although all attachments and select fields were converted to long text    
#### so it needs some manual tweaks    
####    
#### but the table came back alive )   
####    
#### thanks so much!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974412465737711626)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
